src/ert/scheduler/lsf_driver.py

```diff
@@ -103,6 +103,21 @@ def parse_bjobs(bjobs_output: str) -> Dict[str, Dict[str, Dict[str, str]]]:
     return {"jobs": data}
 
 
+def get_missing_jobids_from_bjobs_output(
+    jobs: dict[str, tuple[int, AnyJob, int]], bjobs_states: _Stat
+) -> set[str]:
+    current_timestamp = int(time.time())
+    expected_jobids = list()
+    for job_id, job_data in jobs.items():
+        submitted_timestamp = job_data[2]
+        if current_timestamp < submitted_timestamp + _POLL_PERIOD:
+            continue
+        expected_jobids.append(job_id)
+
+    missing = set(expected_jobids) - set(bjobs_states.jobs.keys())
+    return missing
```

Comment:
I am unsure whether I would to the set substraction inside this function and not outside. If you manage to do the filtering with a list comprehension you might not need this function.

Reply:
Yes, I made it inline instead :)